source/adiabatic_conditions/compute_profile.cc

```diff
@@ -284,6 +306,21 @@ namespace aspect
                              "profile. The higher the number of points, the more accurate "
                              "the downward integration from the adiabatic surface "
                              "temperature will be.");
+          prm.declare_entry ("Use surface condition function", "false",
+                             Patterns::Bool(),
+                             "Whether to use the 'Surface condition function' to determine surface "
+                             "conditions, or the 'Adiabatic surface temperature' and 'Surface pressure' "
+                             "parameters. If this is set to true the reference profile is updated "
+                             "every timestep. The function expression of the function should be "
+                             "independent of space, but can depend on time 't'. The function must "
```

Comment:
in the header you comment that it is ``depth->p,t``, which one is correct?

Reply:
This one is correct, I changed the header.